Exiled.API/Features/Respawn.cs

```diff
@@ -23,6 +23,20 @@ public static class Respawn
         /// </summary>
         public static RespawnEffectsController Controller => RespawnEffectsController.AllControllers.Where(controller => controller != null).FirstOrDefault();
 
+        /// <summary>
+        /// Forces a reinforcement respawn.
+        /// </summary>
+        /// <param name="team">The team to respawn.</param>
+        /// <param name="spawnVehicle">Determines whether or not to spawn the vehicle as well as the team.</param>
+        public static void ForceSpawnTeam(SpawnableTeamType team, bool spawnVehicle = false)
```

Comment:
I would simply call it `Force(...)`, to call it by doing `Respawn.Force`.
You can use the `Respawn.SummonChaosInsurgencyVan` method instead of `ExecuteAllEffects`.

Reply:
I feel like `Respawn.Force` is too simple of a name. That would be like having `Respawn.Summon` instead of `Respawn.SummonChaosInsurgencyVan`

As of the method, I think it's good how it is because I don't want to have to add more code to use the methods individually based on the team.

Reply:
I'd call it `ForceSpawn` without the `Team` postfix, it might be clearer to others, also rename the second parameter to `executeSpawnEffect` (the main game devs call it the `effect`) or something, because `spawnVehicle` is more suitable for CI, but not for NTF.

Reply:
Yeah, but `Respawn.ForceSpawn` sounds like a repetition, so we should name it like `Respawn.ForceWave` or something similar.

Reply:
Literally lost change. Marked as resolved.